frontend/pages/recipe/_slug/index.vue

```diff
@@ -175,72 +175,95 @@
+              <span> {{ $t('recipe.edit-scale') }} </span>
             </v-tooltip>
 
             <template v-if="!recipe.settings.disableAmount && !form">
-              <v-btn color="secondary darken-1" class="mx-1" small @click="scale > 1 ? scale-- : null">
-                <v-icon>
-                  {{ $globals.icons.minus }}
-                </v-icon>
-              </v-btn>
-              <v-btn color="secondary darken-1" small @click="scale++">
-                <v-icon>
-                  {{ $globals.icons.createAlt }}
-                </v-icon>
-              </v-btn>
+              <v-tooltip top color="secondary darken-1">
+                <template v-slot:activator="{ on, attrs }">
+                  <v-btn color="secondary darken-1" class="mx-1" small v-on="on" @click="scale > 1 ? scale-- : null">
+                    <v-icon>
+                      {{ $globals.icons.minus }}
+                    </v-icon>
+                  </v-btn>
+                </template>
+                <span> Decrease Scale by 1 </span>
+              </v-tooltip>
+              <v-tooltip top color="secondary darken-1">
+                <template v-slot:activator="{ on, attrs }">
+                  <v-btn color="secondary darken-1" small v-on="on" @click="scale++">
+                    <v-icon>
+                      {{ $globals.icons.createAlt }}
+                    </v-icon>
+                  </v-btn>
+                </template>
+                <span> Increase Scale by 1 </span>
+              </v-tooltip>
             </template>
             <v-spacer></v-spacer>
 
-            <RecipeRating
-              v-if="enableLandscape && $vuetify.breakpoint.smAndUp"
-              :key="recipe.slug"
-              :value="recipe.rating"
-              :name="recipe.name"
-              :slug="recipe.slug"
-            />
+            <RecipeRating v-if="enableLandscape && $vuetify.breakpoint.smAndUp" :key="recipe.slug"
+              :value="recipe.rating" :name="recipe.name" :slug="recipe.slug" />
           </div>
 
           <v-row>
             <v-col cols="12" sm="12" md="4" lg="4">
-              <RecipeIngredients
-                v-if="!form"
-                :value="recipe.recipeIngredient"
-                :scale="scale"
-                :disable-amount="recipe.settings.disableAmount"
-              />
+              <RecipeIngredients v-if="!form" :value="recipe.recipeIngredient" :scale="scale"
+                :disable-amount="recipe.settings.disableAmount" />
 
               <!-- Recipe Tools Display -->
               <div v-if="!form && recipe.tools && recipe.tools.length > 0">
                 <h2 class="mb-2 mt-4">Required Tools</h2>
                 <v-list-item v-for="(tool, index) in recipe.tools" :key="index" dense>
+<<<<<<< HEAD
+                  <v-checkbox v-model="recipe.tools[index].onHand" hide-details class="pt-0 my-auto py-auto"
+                    color="secondary" @change="updateTool(recipe.tools[index])">
+=======
```

Comment:
I think you've got some merge artifacts here

Reply:
Ah damn yeah I did a hunk staging/commit and it seems like I've not staged the merge part, is now fixed